resources/views/lessonResult.blade.php

```diff
@@ -0,0 +1,37 @@
+<?php
+    //Processing user answers data to display.
+    $result = session()->get('result');
+    $success = (in_array('1', $result)) ? array_count_values($result)[1] : 0;
+    extract($result, EXTR_PREFIX_INVALID, 'answer');
+?>
```

Comment:
it's better that controller will handle this code so blade will be cleaner.

Reply:
@jeremiahC 
I could not help using extract function in the view, but others are removed.